CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Main (unreleased)
 
 - Configure the agent to report the use of feature flags to grafana.com. (@marctc)
 
+- Add `--platforms` flag to agentctl cloud-config command. (@tpaschalis)
```

Comment:
Heads up: `agentctl cloud-config` is left undocumented so we tend to not mention it in the changelog. Up to you if you want to mention it, but I've historically left it as an internal detail of our cloud installation process so that the changelog and other things can be OSS focused.

Reply:
You're right, makes more sense as an internal detail. Removing the changelog entry.